performance/falloutPerformanceTest.yaml

```diff
@@ -0,0 +1,183 @@
+# branch and version of the cassandra-extension to benchmark
+branch: "master"
+version: "1.0.0-SNAPSHOT"
+
+# Whether or not the DataStax Java driver should use the Netty event loop provided by the Quarkus framework
+useQuarkusNettyEventLoop: "true"
+
+# Whether or not the application that is under the test should run in the native mode
+native: "true"
+
+---
+
+ensemble:
+  server:
+    node.count: 3
+    provisioner:
+      name: ctool
+      properties:
+        cloud.provider: openstack
+        cloud.tenant: performance
+        cloud.instance.type: ms1.small
+        name: dseserver-cassandra-quarkus-perf
+        mark_for_reuse: true
+    configuration_manager:
+      - name: ctool
+        properties:
+          java.version: "1.8_151"
+          product.type: dse
+          product.install.type: tarball
+          product.version: 6.7-dev
```

Comment:
I wonder if OSS Cassandra isn't enough?

Reply:
If we would like to test graph or continuous paging in the future maybe it is better to use DSE